pkg/services/accesscontrol/database/database.go

```diff
@@ -45,6 +48,9 @@ type AccessControlStore struct {
 }
 
 func (s *AccessControlStore) GetUserPermissions(ctx context.Context, query accesscontrol.GetUserPermissionsQuery) ([]accesscontrol.Permission, error) {
+	ctx, span := tracer.Start(ctx, "accesscontrol.database.GetUserPermissions")
```

Comment:
question: do you think it's worth adding some query attributes to the trace?

Reply:
It could be. Are you thinking number of permissions / filtering etc?

Reply:
this might be useful but I think it might be better to add these in more targeted PRs later - sometimes it's not obvious up-front what's missing, until you're staring at a span thinking "gee I wish xyz was an attribute here!" 😉

Reply:
That's fair, let's extend it when we feel the need for it.